frame/asset-conversion/src/benchmarking.rs

```diff
@@ -23,20 +23,18 @@ use frame_support::{
 	assert_ok,
 	storage::bounded_vec::BoundedVec,
 	traits::{
-		fungible::{Inspect as InspectFungible, Unbalanced},
+		fungible::{Inspect as InspectFungible, Mutate as MutateFungible, Unbalanced},
```

Comment:
Where is the `MutateFungible` alias used?

Reply:
in `T::Currency::set_balance()`